ga4gh/cli.py

```diff
@@ -243,26 +241,11 @@ def addGlobalOptions(parser):
         "--port", "-P", default=8000, type=int,
         help="The port to listen on")
     parser.add_argument(
-        "--config", "-C", default='DefaultConfig', type=str,
+        "--config", "-c", default='DevelopmentConfig', type=str,
```

Comment:
I think we usually fill up the lowercase letters first when making options. I'd imagine -f is a bit more common for filename input than -F. I'm happy to change it back if you don't like it, I'm not that bothered one way or the other.